src/activities/components/ActivityItem.vue

```diff
@@ -134,9 +134,30 @@
         flat
         no-caps
         align="between"
+        type="a"
         color="secondary"
         :class="{ 'full-width': $q.platform.is.mobile }"
-        class="open-conversation-button"
+        class="activity-item-button"
+        :href="`/api/activities-ics/${activity.id}/`"
+      >
+        <template #default>
+          <div>
+            <QIcon
+              name="fas fa-calendar-plus"
+              size="xs"
+              class="q-mr-xs icon-left"
+            />
+            <span>{{ $t('ACTIVITYLIST.ITEM.EXPORT_ICS') }}</span>
```

Comment:
I wonder if this button is a bit _too_ prominent, given that I think ics export is likely to be a niche feature for most of the users. But I don't have any bright ideas of how to make it less prominent... maybe @larzon83 has some input on this?

Screenshot for convenience (as no branch deployment...):

![activityitem](https://user-images.githubusercontent.com/31616/113412404-102f6200-93b0-11eb-80d8-8c2152667f9f.png)

Reply:
yeah i have no idea what i am doing you know - its just a random place i picked. total happy with other solutions

Reply:
In my opinion I think it would look better without the text and just with the icon. The text "Export to ICS" could be a hover just like the ones we use now for the activity type icon.

Nice little feature btw, I will probably use it!

Reply:
I like the suggestion from @brnsolikyl and unless @larzon83 has any other ideas, seems good to go! Maybe wait a couple of days and see if anybody else bites...

Reply:
not sure about the icon only and how it then looks on mobile. I'll think about it. Give me a few days...

Reply:
Maybe like this? The same could be then used on mobile, too -> no more stretching the button to full width there.

<img width="1454" alt="idea-ics-btn" src="https://user-images.githubusercontent.com/23213583/114104513-4c405680-98cb-11eb-8145-4d83e757263d.png">

I just fiddled in devTools, so please ignore the wrong icon and the mix of german and english 😃 

I would only display this button for joined activities. Or why would I want to download an ics for an event I don't participate?
Another benefit would be, that it would make the button "pop" after joining an activity, making users aware of that functionality.

Also, I would label it "Download ICS". Because for a user that's what it is in the end.

Reply:
I like it showing after you join the activity, makes more sense.

Reply:
I like the idea!

Reply:
Button is now only visible when the user has joined the activity.